packages/css/src/components/description-list/description-list.scss

```
@@ -54,8 +62,22 @@
   }
 }
 
-@mixin reset-dd {
-  margin-inline: 0;
+.ams-description-list__section {
+  @media screen and (min-width: $ams-breakpoint-medium) {
+    column-gap: var(--ams-description-list-column-gap);
+    display: grid;
+    grid-column: span 2;
+
+    dd:only-of-type {
```

Let’s target our classes instead of the element names.

```suggestion
    .ams-description-list__description:only-of-type {
```

---

This won't work, `:only-of-type` applies to the tag not the classname.

---

I don’t think it does – yes it selects elements by their tag name, but that doesn’t mean it can’t work together with a class selector.